lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -316,6 +316,13 @@
   window.webkitStorageInfo.PERSISTENT // FAIL
 }
 
+function isOnHttps() {
+  // Check blob URL.
+  const img = document.createElement('img');
+  img.src = URL.createObjectURL(blob);
```

Comment:
Oops. Yea the smoketest wasn't testing anything. Bug prevented the image from being aded to the DOM.
Will fix.